components/ConvertForm/useConvertInputs.js

```diff
@@ -89,8 +86,11 @@ export function useConvertInputs(otherSymbol, toBonded = true) {
     const maxSlippagePct = 1 //slippage hardcoded at 1% for now, can be made changeable in the future
 
     const estReceived = amount.mul(100 - applicableTribute).div(100)
+    const amountRetained = amount.mul(applicableTribute).div(100) 
     const receivedWithSlippage = estReceived.mul(100 - maxSlippagePct).div(100)
 
+    const singleUnit = (amountSource.isZero()  ? 0 : estReceived/amountSource)
```

Comment:
Even if `amountSource = 0` we still should be able to fetch the price from `useBondingCurvePrice`. Could you try that, please?